src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -544,58 +512,66 @@ export default class LazyMobXTable<T> extends React.Component<LazyMobXTableProps
         this.store.setProps(nextProps);
     }
 
-    buildPaginationControls(className:string | undefined, style: {[k:string]:string | number}, textBetweenButtons:string ): JSX.Element {
-        return <PaginationControls
-            className={className}
-            itemsPerPage={this.store.itemsPerPage}
-            currentPage={this.store.page}
-            onChangeItemsPerPage={this.handlers.changeItemsPerPage}
-            onPreviousPageClick={this.handlers.decPage}
-            onNextPageClick={this.handlers.incPage}
-            textBetweenButtons={textBetweenButtons}
-            previousPageDisabled={this.store.page === 0}
-            style={style}
-            nextPageDisabled={this.store.page === this.store.maxPage}
-            {...this.props.paginationProps}
-        />
+    @computed get paginationControls() {
+        return (
+            <PaginationControls
+                className="pull-left topPagination"
+                itemsPerPage={this.store.itemsPerPage}
+                currentPage={this.store.page}
+                onChangeItemsPerPage={this.handlers.changeItemsPerPage}
+                onPreviousPageClick={this.handlers.decPage}
+                onNextPageClick={this.handlers.incPage}
+                textBetweenButtons={this.store.paginationStatusText}
+                previousPageDisabled={this.store.page === 0}
+                nextPageDisabled={this.store.page === this.store.maxPage}
+                {...this.props.paginationProps}
+            />
+        );
+    }
+
+    @computed get toolbar() {
```

Comment:
I see, you mean as opposed to moving this stuff out into a separate Observer component. That sounds like a good idea to me